### PR TITLE
Crystalline crag

### PR DESCRIPTION
**Pull request by sLackeR-hub**
Crystalline Crag and includes all quests/dungeons associated with it. (Excluding Exploration Markers)

### CODE REVIEW

**Review by ziang4891 — CHANGES_REQUESTED** [review]
The files within 9 WeenieDefaults shouldn't be just loose within this base folder but should be sorted into subfolders that follow the convention of WeenieType/ItemType

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/39984 Adventurer's Token.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
+DELETE FROM `weenie` WHERE `class_Id` = 39984;
+
+INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
+VALUES (39984, 'ace39984-adventurerstoken', 51, '2019-02-10 00:00:00') /* Stackable */;
+
```
Comment (ziang4891) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/39984 Adventurer's Token.sql:5]: This weenie file already exists in a previous patch folder, and there are not differences between the two, so a duplicate is not needed in the 2009-02-UnfinishedBusiness patch directory tree

**Review by ziang4891 — CHANGES_REQUESTED** [review]

**Comment by ziang4891** [comment]
The creatures that don't have a PropertyInt.CreatureType should be placed in Creature\Unsorted subfolder tree

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40100 Crystalline Shard.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (40100,   1, 'Crystalline Shard') /* Name */
+     , (40100,  16, 'A crystalline shard condensed from the remains of a crystalline wisp.') /* LongDesc */
+	  , (40100,   20, 'Crystalline Shards') /* Name */;
+	  
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (40100,   1,   33557506) /* Setup */
+     , (40100,   3,  536870932) /* SoundTable */
+     , (40100,   8,  100690349) /* Icon */
+     , (40100,  22,  872415275) /* PhysicsEffectTable */;
```
Comment (ziang4891) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40100 Crystalline Shard.sql:36]: Use the file from ACE-PCAP-Dynamics-Exports, as it has the correct WeenieType and property composition

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40103 Oorjit.sql [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+     , (40103,   3,  536870913) /* SoundTable */
+     , (40103,   6,   67108990) /* PaletteBase */
+     , (40103,   8,  100667446) /* Icon */
+     , (40103,   9,   83890505) /* EyesTexture */
+     , (40103,  10,   83890530) /* NoseTexture */
+     , (40103,  11,   83890666) /* MouthTexture */
+     , (40103,  15,   67117019) /* HairPalette */
+     , (40103,  16,   67109567) /* EyesPalette */
+     , (40103,  17,   67109557) /* SkinPalette */;
+
```
Comment (ziang4891) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40103 Oorjit.sql:43]: The fixed Texture DIDs should be removed to allow for randomization

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40099 Tokashi Ando.sql [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+     , (40099,   3,  536870913) /* SoundTable */
+     , (40099,   6,   67108990) /* PaletteBase */
+     , (40099,   8,  100667446) /* Icon */
+     , (40099,   9,   83890447) /* EyesTexture */
+     , (40099,  10,   83890521) /* NoseTexture */
+     , (40099,  11,   83890641) /* MouthTexture */
+     , (40099,  15,   67117075) /* HairPalette */
+     , (40099,  16,   67110063) /* EyesPalette */
+     , (40099,  17,   67110054) /* SkinPalette */;
+
```
Comment (ziang4891) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40099 Tokashi Ando.sql:44]: The fixed Texture DIDs should be removed to allow for randomization

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40097 Jyotsana.sql [hunk 5]
```diff
@@ -0,0 +1,415 @@
...
+     , (40097,   2,  150994945) /* MotionTable */
+     , (40097,   3,  536870914) /* SoundTable */
+     , (40097,   6,   67108990) /* PaletteBase */
+     , (40097,   8,  100667446) /* Icon */
+     , (40097,   9,   83890261) /* EyesTexture */
+     , (40097,  10,   83890288) /* NoseTexture */
+     , (40097,  11,   83890352) /* MouthTexture */
+     , (40097,  15,   67116994) /* HairPalette */
+     , (40097,  16,   67110063) /* EyesPalette */
+     , (40097,  17,   67109554) /* SkinPalette */;
```
Comment (ziang4891) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40097 Jyotsana.sql:42]: The fixed Texture DIDs should be removed to allow for randomization

**Review by ziang4891 — CHANGES_REQUESTED** [review]

**Comment by ziang4891** [comment]
PropertyBools, like IgnoreCollisions, Ethereal, GravityStatus, and the like are not needed and can be removed with there is a matching PropertyInt.PhysicsState bitfield that contains the same properties

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/39984 Adventurer's Token.sql (hunk 1 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/39984 Adventurer's Token.sql]: Removed/

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40097 Jyotsana.sql (hunk 5 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40097 Jyotsana.sql]: Removed DIDs

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40099 Tokashi Ando.sql (hunk 4 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40099 Tokashi Ando.sql]: Removed DIDs

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40103 Oorjit.sql (hunk 3 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40103 Oorjit.sql]: Removed DIDs.

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40100 Crystalline Shard.sql (hunk 2 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/40100 Crystalline Shard.sql]: Fixed.

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Crystal/40106 Crystal.sql [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+     , (40106, 379,          0) /* GearMaxHealth */
+     , (40106, 381,          0) /* PKDamageRating */
+     , (40106, 382,          0) /* PKDamageResistRating */
+     , (40106, 383,          0) /* GearPKDamageRating */
+     , (40106, 384,          0) /* GearPKDamageResistRating */
+     , (40106, 386,          0) /* Overpower */
+     , (40106, 387,          0) /* OverpowerResist */
+     , (40106, 388,          0) /* GearOverpower */
+     , (40106, 389,          0) /* GearOverpowerResist */;
+
```
Comment (ziang4891) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Crystal/40106 Crystal.sql:37]: All of these ratings, especially when they are zero, should be removed from this weenie and others

**Review by ziang4891 — CHANGES_REQUESTED** [review]

**Comment by ziang4891** [comment]
If you move 40124 Crystal.sql to Generic/Misc and remove the PropertyInt ratings that are 0 from a couple of the Crystal NPCs, I think this PR should be good

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Crystal/40106 Crystal.sql (hunk 6 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Crystal/40106 Crystal.sql:13]: Fixed.

**Review by ziang4891 — CHANGES_REQUESTED** [review]
All of the Crystal NPCs that are not attackable should have the AiImmobile and NpcLooksLikeObject bools, as they are Creatures for the purposes of performing game mechanics but appraise as objects not creatures

**Comment by sLackeR-hub** [comment]
> 
> 
> All of the Crystal NPCs that are not attackable should have the AiImmobile and NpcLooksLikeObject bools, as they are Creatures for the purposes of performing game mechanics but appraise as objects not creatures

**Comment by ziang4891** [comment]
Looks good.  I'll need to load it into my server to give it a test drive.

**Review by ziang4891 — CHANGES_REQUESTED** [review]
71868 Crystalline Array Fragment Generator.sql file shouldn't be in the **\B GameEventDefDB** folder but should be in **\9 WeenieDefaults\Generic\None**

**Review by ziang4891 — APPROVED** [review]

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Wisp/42670 Shivering Crystalline Wisp.sql [hunk 7]
```diff
@@ -0,0 +1,116 @@
+DELETE FROM `weenie` WHERE `class_Id` = 42670;
+
+INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
+VALUES (42670, 'ace42670-shiveringcrystallinewisp', 10, '2019-02-10 00:00:00') /* Creature */;
+
```
Comment (ziang4891) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Wisp/42670 Shivering Crystalline Wisp.sql:5]: This file should not be here and shouldn't be used for the Crystalline Crag data.  This wcid of the wisp belongs to the Aetheria dungeons

**Review by ziang4891 — CHANGES_REQUESTED** [review]

**Review by ziang4891 — DISMISSED** [review]

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Wisp/42670 Shivering Crystalline Wisp.sql (hunk 7 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Wisp/42670 Shivering Crystalline Wisp.sql]: Removed.

**Comment by DridsGhost** [comment]
316D looks like it has mangled data from 01970 High Direlands Mountain Generator.sql

You should pull out this data and clean up the landblock.

**Comment by DridsGhost** [comment]
WeenieClassName: ace40104-resonantportal shows as a creature in the PCAP.

Why is this a white portal

**Comment by DridsGhost** [comment]
All of your landblocks appear to use the generic dynamics linkables on 14 minute spawns. 01154.

Should they be that long?

**Comment by DridsGhost** [comment]
`landblock` = 0x326A

More garbled with encounters gens

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Unsorted/40104 Resonant Portal.sql [hunk 8]
```diff
@@ -13,7 +13,8 @@ VALUES (40104,   1,         16) /* ItemType - Creature */
      , (40104, 133,          4) /* ShowableOnRadar - ShowAlways */;
 
 INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
-VALUES (40104,   1, True ) /* Stuck */;
+VALUES (40104,   1, True ) /* Stuck */
+	  , (40104,  83, True ) /* NPCLooklike */;
```
Comment (DridsGhost) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Unsorted/40104 Resonant Portal.sql:17]: This weenie should be a creature according to the PCAP

Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Unsorted/40104 Resonant Portal.sql]: Fixed

File: Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/2D6B.sql [hunk 9]
```diff
@@ -0,0 +1,603 @@
...
+INSERT INTO `landblock_instance` (`guid`, `weenie_Class_Id`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`, `is_Link_Child`, `last_Modified`)
+VALUES (0x72D6B06A,  4455, 0x2D6B02FB, 190.1, 159, 38.4, -0.707107, 0, 0, -0.707107, False, '2019-02-10 00:00:00'); /* Door */
+/* @teleloc 0x2D6B02FB [190.100000 159.000000 38.400000] -0.707107 0.000000 0.000000 -0.707107 */
+
+INSERT INTO `landblock_instance` (`guid`, `weenie_Class_Id`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`, `is_Link_Child`, `last_Modified`)
+VALUES (0x72D6B06B,  1154, 0x2D6B0036, 165.4855, 126.6976, 55.77747, -0.6962002, 0, 0, -0.7178476, False, '2019-02-10 00:00:00'); /* Linkable Monster Generator */
+/* @teleloc 0x2D6B0036 [165.485500 126.697600 55.777470] -0.696200 0.000000 0.000000 -0.717848 */
+
+INSERT INTO `landblock_instance_link` (`parent_GUID`, `child_GUID`, `last_Modified`)
+VALUES (0x72D6B06B, 0x72D6B06C, '2019-02-10 00:00:00') /* Acidic Diamond Golem (36829) */
```
Comment (DridsGhost) [inline Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/2D6B.sql:208]: This mob shouldnt be here

**Comment by DridsGhost** [comment]
http://acpedia.org/wiki/Crystalline_Adventurer

Crystalline Shards
Kill various wisps throughout the Crystalline Crag and its dungeons.
Wisps will randomly drop a yellow-colored Crystalline Shard.
Hand them to Tokashi Ando for 50,000 xp each.
Note: There is no pickup timer and they are giveable. Any character of level 50+ can hand them in for the xp.
Note: You can only turn in 20 shards within a 20 hour period.

This quest excepts unlimited Crystalline Shards

--------------------------

**Comment by DridsGhost** [comment]
Crystalline Bound Wisp

Currently it says
Bound Wisp tells you, "Excellent work in your hunt. We'll find your reports very useful."

It should be a DirectBroadcast: You have reached the bound wisp!

https://youtu.be/_XgTlViV6WE?t=218

**Comment by DridsGhost** [comment]
Crystalline Essence Array should be default Scale of 2

**Comment by DridsGhost** [comment]
Bound Wisp should cast 

Bound Wisp teleports you with Crystalline Portal.

Change dialouge to 
DirectBroadcast:
You have reached the bound wisp!

**Comment by DridsGhost** [comment]
Landblock 326A

Needs to be cleaned up

**Comment by DridsGhost** [comment]
Landblock 72E6 needs to be cleaned up

There are torches everywhere  not mentioned in the wiki. Likely a player drop.

Other various landscape creatures, supply crates, lugians etc..

**Comment by ziang4891** [comment]
Don't leave the static spawns of plants, either

**Comment by DridsGhost** [comment]
Looks like we are missing contracts. Also CrystallineArrayFragmentPickupTimer.sql should be CrystalFinalPickup0509

And CrystallineMarkerQuest.sql should be CrystallineMarkerCounter0109

For the contracts to work.

Tokashi Ando			CrystalFinalPickup0509
Bound Wisp		       CrystallineSkillTest10109Wait
Oorjit				KillTaskCrystallineWisps10109		KillTaskCrystallineWisps10109Wait
Jyotsana				CrystallineMarkerCounter0109

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Golem/40775 Ancient Golem.sql [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+     , (40775, 122,       2) /* AiAcquireHealth */
+     , (40775, 123,       2) /* AiAcquireStamina */
+     , (40775, 124,       2) /* AiAcquireMana */
+     , (40775, 125,       1) /* ResistHealthDrain */;
+
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (40775,   1, 'Ancient Golem') /* Name */;
+
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (40775,   1,   33559702) /* Setup */
```
Comment (DridsGhost) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Golem/40775 Ancient Golem.sql:70]: These guys have the wrong values. Should be

SETUP_DID (1): 0x02001496 (33559702)
CLOTHINGBASE_DID (7): 0x10000637 (268437047)
PALETTE_TEMPLATE_INT (3): 39 - Black
SHADE_FLOAT (12): 0

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Golem/40775 Ancient Golem.sql [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+     , (40775,   2,  150995344) /* MotionTable */
+     , (40775,   3,  536870933) /* SoundTable */
+     , (40775,   4,  805306376) /* CombatTable */
+     , (40775,   6,   67116739) /* PaletteBase */
+     , (40775,   8,  100667940) /* Icon */
+     , (40775,  22,  872415332) /* PhysicsEffectTable */
+     , (40775,  35,       1026) /* DeathTreasureType - Loot Tier: 7 */;
```
Comment (DridsGhost) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Golem/40775 Ancient Golem.sql:77]: These golems should be T6. Treasure type 

461) /* DeathTreasureType - Loot Tier: 6 */;

**Review by DridsGhost — CHANGES_REQUESTED** [review]

File: Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/2D6B.sql [hunk 12]
```diff
@@ -0,0 +1,578 @@
...
+INSERT INTO `landblock_instance` (`guid`, `weenie_Class_Id`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`, `is_Link_Child`, `last_Modified`)
+VALUES (0x72D6B0A0, 40096, 0x2D6B0104, 136, 169, -75.6, 1, 0, 0, 0,  True, '2019-02-10 00:00:00'); /* Crystal */
+/* @teleloc 0x2D6B0104 [136.000000 169.000000 -75.600000] 1.000000 0.000000 0.000000 0.000000 */
+
+INSERT INTO `landblock_instance` (`guid`, `weenie_Class_Id`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`, `is_Link_Child`, `last_Modified`)
+VALUES (0x72D6B0A1, 40093, 0x2D6B019D, 143.586, 71.277, -3.6, 0.408487, 0, 0, -0.912764,  True, '2019-02-10 00:00:00'); /* Crystal */
+/* @teleloc 0x2D6B019D [143.586000 71.277000 -3.600000] 0.408487 0.000000 0.000000 -0.912764 */
+
+INSERT INTO `landblock_instance` (`guid`, `weenie_Class_Id`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`, `is_Link_Child`, `last_Modified`)
+VALUES (0x72D6B0A2,  1542, 0x2D6B002F, 137.0873, 159.8326, 68.43377, 0.9980775, 0, 0, -0.06197855, False, '2019-02-10 00:00:00'); /* Linkable Item Generator */
```
[the remark this answers is not in the document]
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/2D6B.sql:466]: Switched to 2 min gen

File: Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/00C7.sql [hunk 13]
```diff
@@ -0,0 +1,1576 @@
...
+INSERT INTO `landblock_instance` (`guid`, `weenie_Class_Id`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`, `is_Link_Child`, `last_Modified`)
+VALUES (0x700C7289, 40106, 0x00C70219, 100.357, -79.728, -17.95, 0.770949, 0, 0, -0.636897,  True, '2019-02-10 00:00:00'); /* Crystal */
+/* @teleloc 0x00C70219 [100.357000 -79.728000 -17.950000] 0.770949 0.000000 0.000000 -0.636897 */
+
+INSERT INTO `landblock_instance` (`guid`, `weenie_Class_Id`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`, `is_Link_Child`, `last_Modified`)
+VALUES (0x700C728A, 40107, 0x00C70100, 101.4883, -78.15381, -41.95, 0.995649, 0, 0, 0.093185,  True, '2019-02-10 00:00:00'); /* Crystal */
+/* @teleloc 0x00C70100 [101.488300 -78.153810 -41.950000] 0.995649 0.000000 0.000000 0.093185 */
+
+INSERT INTO `landblock_instance` (`guid`, `weenie_Class_Id`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`, `is_Link_Child`, `last_Modified`)
+VALUES (0x700C728B,  1542, 0x00C70390, 100, -70, 0, 1, 0, 0, 0, False, '2019-02-10 00:00:00'); /* Linkable Item Generator */
```
[the remark this answers is not in the document]
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/00C7.sql:1458]: I guess I don't know either. These are the crystals that spawn the "rare" wisps so 13 should be fine? Let me know if anyone thinks I should change it.

File: Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/326B.sql [hunk 14]
```diff
@@ -0,0 +1,223 @@
...
+INSERT INTO `landblock_instance_link` (`parent_GUID`, `child_GUID`, `last_Modified`)
+VALUES (0x7326B001, 0x7326B002, '2019-02-10 00:00:00') /* Incalescent Crystalline Wisp (40131) */
+     , (0x7326B001, 0x7326B003, '2019-02-10 00:00:00') /* Incalescent Crystalline Wisp (40131) */
+     , (0x7326B001, 0x7326B004, '2019-02-10 00:00:00') /* Crystalline Wisp (40134) */
+     , (0x7326B001, 0x7326B005, '2019-02-10 00:00:00') /* Incalescent Crystalline Wisp (40131) */
+     , (0x7326B001, 0x7326B006, '2019-02-10 00:00:00') /* Shivering Crystalline Wisp (40127) */
+     , (0x7326B001, 0x7326B007, '2019-02-10 00:00:00') /* Crystalline Wisp (40134) */
+     , (0x7326B001, 0x7326B008, '2019-02-10 00:00:00') /* Corroding Crystalline Wisp (40128) */
+     , (0x7326B001, 0x7326B009, '2019-02-10 00:00:00') /* Crystalline Wisp (40130) */
+     , (0x7326B001, 0x7326B00A, '2019-02-10 00:00:00') /* Pyreal Golem (36833) */
```
[the remark this answers is not in the document]
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/326B.sql:16]: Removed.

File: Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/326B.sql [hunk 15]
```diff
@@ -0,0 +1,223 @@
...
+     , (0x7326B001, 0x7326B01D, '2019-02-10 00:00:00') /* Incalescent Crystalline Wisp (40131) */
+     , (0x7326B001, 0x7326B01E, '2019-02-10 00:00:00') /* Corroding Crystalline Wisp (40128) */
+     , (0x7326B001, 0x7326B01F, '2019-02-10 00:00:00') /* Crystalline Wisp (40134) */
+     , (0x7326B001, 0x7326B020, '2019-02-10 00:00:00') /* Corroding Crystalline Wisp (40128) */
+     , (0x7326B001, 0x7326B021, '2019-02-10 00:00:00') /* Incalescent Crystalline Wisp (40131) */
+     , (0x7326B001, 0x7326B022, '2019-02-10 00:00:00') /* Corroding Crystalline Wisp (40128) */
+     , (0x7326B001, 0x7326B023, '2019-02-10 00:00:00') /* Crystalline Wisp (40130) */
+     , (0x7326B001, 0x7326B024, '2019-02-10 00:00:00') /* Corroding Crystalline Wisp (40128) */
+     , (0x7326B001, 0x7326B025, '2019-02-10 00:00:00') /* Incalescent Crystalline Wisp (40131) */
+     , (0x7326B001, 0x7326B026, '2019-02-10 00:00:00') /* Gotrok Tiatus (24497) */
```
[the remark this answers is not in the document]
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/326B.sql:44]: Removed.

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Generic/Misc/71867 Crystalline Array Fragment.sql [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+INSERT INTO `weenie_properties_float` (`object_Id`, `type`, `value`)
+VALUES (71867,  39, 0.3) /* DefaultScale */;
+
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (71867,   1, 'Crystalline Array Fragment') /* Name */
+     , (71867,  16, 'A fragment of a crystalline array destroyed after releasing it using the Empyrean essence lock devices.') /* LongDesc */
+     , (71867,  33, 'CrystalFinalPickup0509') /* Quest */;
+     
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (71867,   1,   33557506) /* Setup */
```
[the remark this answers is not in the document]
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Generic/Misc/71867 Crystalline Array Fragment.sql:31]: Fixed, is white now.

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Golem/40775 Ancient Golem.sql (hunk 10 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Golem/40775 Ancient Golem.sql:70]: Fixed

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Golem/40775 Ancient Golem.sql (hunk 11 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Creature/Golem/40775 Ancient Golem.sql:77]: Fixed.

File: Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/00C7.sql (hunk 13 above)
[the remark this answers is not in the document]
Comment (DridsGhost) [reply Database/Patches/2009-02-UnfinishedBusiness/6 LandBlockExtendedData/00C7.sql:1458]: Fine by me. We can wait for it to go live and adjust if necessary.

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Generic/Misc/71867 Crystalline Array Fragment.sql [hunk 17]
```diff
@@ -1,10 +1,11 @@
 DELETE FROM `weenie` WHERE `class_Id` = 71867;
 
 INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
-VALUES (71867, 'ace71867-CrystallineArrayFragment', 38, '2020-08-21 23:49:56') /* Generic */;
+VALUES (71867, 'ace71867-CrystallineArrayFragment', 38, '2020-08-21 23:49:56') /* Gem */;
```
Comment (ziang4891) [inline Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Generic/Misc/71867 Crystalline Array Fragment.sql:4]: Why did you change this item to a Gem from Generic?  Generic fits better, given the background of the icon.

Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Generic/Misc/71867 Crystalline Array Fragment.sql:4]: That is a good question I don't recall changing it, but I do see the value remained the same for type so I am guessing the /export updated the comment.

**Review by sLackeR-hub — COMMENTED** [review]
Not sure why this is a review sorry.

File: Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Generic/Misc/71867 Crystalline Array Fragment.sql (hunk 17 above)
Comment (ziang4891) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Generic/Misc/71867 Crystalline Array Fragment.sql:4]: The weenietype value changed from 1 to 38, which is why the comment changed.  It needs to be changed back to a Generic item.  It is not a Gem

Comment (sLackeR-hub) [reply Database/Patches/2009-02-UnfinishedBusiness/9 WeenieDefaults/Generic/Misc/71867 Crystalline Array Fragment.sql]: Fixed/reverted.

**Review by ziang4891 — DISMISSED** [review]

**Review by ziang4891 — APPROVED** [review]

**Comment by ziang4891** [comment]
Will require cumulative inclusion of PRs 3405, 3406, and 3407 from ACE

**Review by ziang4891 — APPROVED** [review]

**Pull request merged**